itou/users/models.py

```diff
@@ -349,6 +350,13 @@ class Meta(AbstractUser.Meta):
     def __str__(self):
         return f"{self.get_full_name()} — {self.email}"
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.__original_birthdate = self.birthdate
+        self.__original_first_name = self.first_name
+        self.__original_last_name = self.last_name
+        self.__original_nir = self.nir
```

Comment:
```suggestion
        self._original_birthdate = self.birthdate
        self._original_first_name = self.first_name
        self._original_last_name = self.last_name
        self._original_nir = self.nir
```
pourrait suffire (et nous éviterait de se battre avec le "name mangling" en debug)

Reply:
J'ai proposé une version alternative qui semble mieux correspondre.